CLI.md

```diff
@@ -0,0 +1,172 @@
+# AR.IO CLI
+
+The AR.IO CLI is a Node.js command line interface for interacting with the AR.IO network. It allows you to perform various actions such as transferring ARIO, delegating stake, buying records, and more. The CLI is designed to be easy to use and provides detailed help information for each command. This document provides an overview of the CLI and its usage.
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Options](#options)
+- [Commands](#commands)
+- [Development](#development)
+
+<!-- tocstop -->
+
+## Installation
+
+Ensure that you have Node.js installed on your system before installing the CLI. You can check if Node.js is installed by running:
+
+```bash
+node --version
+```
+
+The AR.IO CLI can be installed using npm or yarn. To install the CLI, run one the following commands:
+
+```bash
+npm install -g @ar.io/sdk
+```
+
+or
+
+```bash
+yarn global add @ar.io/sdk --ignore-engines
```

Comment:
nit: may want to include note why ignore-engines is needed, reference README.md for why we have it

Reply:
[8ff3f23](https://github.com/ar-io/ar-io-sdk/pull/334/commits/8ff3f234660db477f148c3df68e1d51820d0ec25)